RescheduleAppointments/force-app/main/default/lwc/mobileAppointmentBookingResourceAssignment/mobileAppointmentBookingResourceAssignment.js

```diff
@@ -17,6 +18,13 @@ export default class MobileAppointmentBookingResourceAssignment extends Lightnin
     return this.isExcluded;
   }
 
+  @api get showMobileWorkerChoice() {
+    return this._showMobileWorkerChoice;
+  }
+  set showMobileWorkerChoice(value) {
+    this._showMobileWorkerChoice = value;
+  }
```

Comment:
Just curious: why can't it just be defined for example  the way you've defined `isExcluded` on line 10? Why do you need to define it as a get/set ?

Reply:
@maliroteh-sf Actually I was setting additional things from there and forgot to revet